fastlane/lib/fastlane/plugins/plugin_info_collector.rb

```diff
@@ -72,7 +72,13 @@ def fix_plugin_name(name)
     # Author
     #
 
-    def collect_author
+    def detect_author
+      git_name = Helper.backticks('git config --get user.name').strip
+      return git_name.empty? ? nil : git_name
+    end
+
+    def collect_author(initial_author = nil)
+      return initial_author if !initial_author.nil? && author_valid?(initial_author)
```

Comment:
`author_valid?` can already handle a `nil` input (returning `false`) so I think this can be shortened to 
```ruby
return initial_author if author_valid?(initial_author)
```